src/ape/cli/choices.py

```diff
@@ -36,16 +38,19 @@ def __init__(self, account_type: Optional[Type[AccountAPI]] = None):
         self._account_type = account_type
 
     @property
-    def choices(self) -> List[str]:  # type: ignore
+    def choices(self) -> Iterator[str]:  # type: ignore
```

Comment:
I should maybe use the LazySequence for all these.
We are using iterators here but technically click says they have to be sequences. Things have been working but technically click can assume sequences and our stuff may suddenly be failing. 

Click should consider supporting iterators explicitly, that would make our lives easier. But for now, we have a custom type that matches the Sequence protocol while still being as effective as an iterator in its laziness.